TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/setup.py

```diff
@@ -0,0 +1,26 @@
+from setuptools import setup, find_packages  # ignore: type
+
+setup(
+    name="console_link",
+    version="1.0.0",
+    description="A Python module to create a console application from a Python script",
+    packages=find_packages(exclude=("tests")),
+    install_requires=["requests", "boto3", "pyyaml", "Click", "cerberus"],
+    entry_points={
+        "console_scripts": [
+            "console = console_link.cli:cli",
+        ],
+    },
+    classifiers=[
+        "Development Status :: 3 - Alpha",
+        "Intended Audience :: Developers",
+        "Topic :: Software Development :: Build Tools",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
+        "License :: OSI Approved :: MIT License",
```

Comment:
This repo is `Apache-2.0 license`, pretty should our license should be in sync, no?
What does OSI Approved mean?

Reply:
@gregschohn @lewijacn this needs to be resolved before this PR is merged - who can look into this?

Reply:
Thanks for catching, have updated this to the Apache license